src/test/java/com/slack/kaldb/chunk/ReadWriteChunkImplTest.java

```diff
@@ -315,16 +327,22 @@ public void testCommitBeforeSnapshot() {
     private final Duration commitInterval = Duration.ofSeconds(5 * 60);
     private final Duration refreshInterval = Duration.ofSeconds(5 * 60);
     private Chunk<LogMessage> chunk;
+    private LuceneIndexStoreImpl logStore;
 
     @Before
     public void setUp() throws IOException {
       registry = new SimpleMeterRegistry();
-      LuceneIndexStoreImpl logStore =
+      logStore =
           LuceneIndexStoreImpl.makeLogStore(
               temporaryFolder.newFolder(), commitInterval, refreshInterval, registry);
       chunk = new ReadWriteChunkImpl<>(logStore, "testDataSet");
     }
 
+    @BeforeEach
+    public void cleanup() throws IOException {
```

Comment:
Since we create a fresh log store for every test in a temporary folder do we need this additional cleanup before running every test?

Reply:
Maybe I missed it, but I saw the temporary store being created for every test Class. And then there was no cleanup between two tests within the test class.